packages/core/getUserAvailability.ts

```diff
@@ -129,7 +135,7 @@ export async function getUserAvailability(
     selectedCalendars,
   });
 
-  const bufferedBusyTimes = busyTimes.map((a) => ({
+  const bufferedBusyTimes: EventBusyDetails[] = busyTimes.map((a) => ({
```

Comment:
Assign this type to `EventBusyDetails[]` as that is what it was -> TS doesn't infer correctly here